src/streams/StreamScreen.js

```diff
@@ -104,6 +109,23 @@ class StreamScreen extends PureComponent<Props> {
             secondary
             onPress={() => delay(this.handleEditSubscribers)}
           />
+          <View style={styles.padding}>
+            <Label
+              style={[styles.marginTop, { fontSize: 16, }]}
+              text="Email Address"
+            />
+          </View>
+          <TouchableNativeFeedback
+            onPress={() => this.handleCopy(subscription.email_address)}
+            background={Platform.OS === 'android' ? TouchableNativeFeedback.SelectableBackground() : ''}
+          >
```

Comment:
Taking a quick look at how the web app handles this, it looks like it's also done in the stream settings view, so I think it makes sense to put it here. One thing I noticed is that the web app has a question mark icon with a helpful link to a page ([this one](https://chat.zulip.org/help/message-a-stream-by-email)) that describes what the email address is for:

![image](https://user-images.githubusercontent.com/22248748/77199003-55366e00-6aa5-11ea-9335-402d42c1e8f1.png)

In the mobile app, currently, I don't know of a similar pressable question mark icon with a link to the realm's Help page, but it would definitely be useful here, and potentially elsewhere; @gnprice, what do you think? That Help page is realm-specific, so it would need to be connected to the `accounts` state in Redux to get the current realm and point to the specific help page for that realm. Something reusable would be better than a one-off solution, I think.

Reply:
I think having a reusable component will be good! I've seen it at many places like [this](https://chat.zulip.org/help/manage-inactive-streams) and [this](https://chat.zulip.org/help/create-your-organization-profile) as well (all those are redirects to the respective help page). We need to create another issue maybe for that component then?

About the help page being realm specific - isn't that same for all the realms?